lottie-ios/Classes/AnimatableLayers/LOTLayerView.m

```diff
@@ -319,7 +320,15 @@ - (NSString*)description {
 - (void)_setImageForAsset {
   if (_layerModel.imageAsset.imageName) {
     NSArray *components = [_layerModel.imageAsset.imageName componentsSeparatedByString:@"."];
-    UIImage *image = [UIImage imageNamed:components.firstObject];
+    UIImage *image = nil;
+    if ([LOTContext getRegisterBundle] == [NSBundle mainBundle]) {
+      image = [UIImage imageNamed:components.firstObject];
+    }else{
+      NSBundle *bundle = [LOTContext getRegisterBundle];
+      NSString *bundleName = [[bundle bundlePath] lastPathComponent];
+      image = [UIImage imageNamed:[bundleName stringByAppendingPathComponent:components.firstObject]];
```

Comment:
Can you merge this PR please ?
I can't retrieve image in custom bundle.

Thx a lot ! ;-)